README.md

```diff
@@ -1,38 +1,137 @@
-# react-apollo
+# [React Apollo](http://dev.apollodata.com/react/) [![npm version](https://badge.fury.io/js/react-apollo.svg)](https://badge.fury.io/js/react-apollo) [![Get on Slack](https://img.shields.io/badge/slack-join-orange.svg)](http://www.apollostack.com/#slack)
 
-[![npm version](https://badge.fury.io/js/react-apollo.svg)](https://badge.fury.io/js/react-apollo)
-[![Get on Slack](https://img.shields.io/badge/slack-join-orange.svg)](http://www.apollostack.com/#slack)
-[![Build Status](https://travis-ci.org/apollographql/react-apollo.svg?branch=master)](https://travis-ci.org/apollographql/react-apollo)
-[![Coverage Status](https://coveralls.io/repos/github/apollographql/react-apollo/badge.svg?branch=master)](https://coveralls.io/github/apollographql/react-apollo?branch=master)
```

Comment:
I don’t really like them because sometimes they have false negatives, like a build could be failing on a branch and so the build status goes red. That’s definitely the last thing we want 😣

IMO build status and coverage status badges are pretty weak signals compared to things like commit count, issue count, stars, last commit timing, etc. and they have the possibility of displaying a false negative.

If we want to get a badge that always says “build passing” and 90%+ coverage no matter what the temporary status looks like, I’m down for that 😉

Thoughts?